rest/rest-server/src/test/resources/functionaltests/config/flow_loop_out.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.2"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.2 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.2/schedulerjob.xsd"
```

Comment:
If it is a new workflow definition, please use schema version _3.3_. _3.2_ is not the latest one and we can think it may contain old things which are deprecated.